host-configs/rzansel-blueos_3_ppc64le_ib_p9-xl@nvcc.cmake

```diff
@@ -112,6 +112,8 @@ set(CUDA_SEPARABLE_COMPILATION ON CACHE BOOL "")
 
 set(AXOM_CUDA_ARCH "sm_70" CACHE PATH "")
 
+set(AXOM_USE_CUB ON CACHE BOOL "")
```

Comment:
I don't see a data change in the TPL set.

Did you build a set of TPLs or just modify the existing host-config?

If it is the latter, can you please confirm that:
(a) the TPLs build with your new change
(b) our blueos host-configs continue to pass the CI tests.

Reply:
See 64c1721de2f143bdcc05fd115588677717c6457a and 12c3a07b4a24957a463db255908ece401af58ae1